README.md

```diff
@@ -57,13 +57,13 @@ To get the library from the latest oneAPI release, follow the
 instructions from Intel(R) [oneAPI installation
 guide](https://www.intel.com/content/www/us/en/developer/articles/guide/installation-guide-for-oneapi-toolkits.html).
 
-> **NOTE:** You need to install the Intel(R) oneAPI Base Toolkit to get
+> **NOTE:** You need to install the Intel(R) oneAPI Basekit to get
 >IDP and `dpctl`.
 
 
 ## Conda
 
-To install `dpctl` from the Intel(R) channel on Annaconda*
+To install `dpctl` from the Intel(R) channel on Annaconda
```

Comment:
```suggestion
To install `dpctl` from the Intel(R) channel on Anaconda
```

Cf. https://anaconda.org/

Reply:
This was a pre-existing typo, but this PR seems appropriate set of changes to fix it.

Reply:
Good catch!